src/app/devices/deviceContent/components/digitalTwin/digitalTwinInterfaces.tsx

```diff
@@ -110,35 +208,38 @@ export const DigitalTwinInterfaces: React.FC<DigitalTwinInterfacesProps> = props
                 link={ResourceKeys.settings.questions.questions.documentation.link}
                 tooltip={ResourceKeys.settings.questions.questions.documentation.text}
             />
-
-            <section className="device-detail">
-                {props.dcm && <Label className="dcm-info">{t(ResourceKeys.digitalTwin.dcm, {modelId: props.dcm })}</Label>}
-                {modelContents.length !== 0 &&
-                    <div className="list-detail">
-                        <DetailsList
-                            onRenderItemColumn={renderItemColumn()}
-                            items={modelContents}
-                            columns={getColumns()}
-                            selectionMode={SelectionMode.none}
-                        />
-                    </div>
-                }
-            </section>
+            {props.isDigitalTwinLoading ?
+                <MultiLineShimmer/> :
+                <section className="device-detail">
+                    <h4>{t(ResourceKeys.digitalTwin.steps.first)}</h4>
```

Comment:
accessibility issue: h4 can't used randomly without h1, h2, h3

Reply:
This page does have a h3 which is in headerView. Does it also needs h1 and h2?